test/CMakeLists.txt

```diff
@@ -1,15 +1,8 @@
-if (NOT GC_TEST_ENABLE)
+if (NOT GC_ENABLE_TEST)
     message(STATUS "The tests are not enabled.")
     return()
 endif ()
 
 include(gtest)
-
-# Static graph compiler library to be used in tests
-add_mlir_library(graph_compiler_static STATIC ${GC_LIB_SOURCES})
```

Comment:
Is there a new target for this?

Reply:
Yes, GcDnnlStatic and it's only enabled if GC_ENABLE_TEST_DNNL.